src/capitalsFirst.js

```diff
@@ -19,6 +19,19 @@
  */
 function capitalsFirst(str) {
   // write code here
-}
+  let arr = str.split(' ');
+  let arrUpperCase = [];
+  let arrLowerCase = [];
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
Give preference to the array methods over the for loop
Better solution 1: do the same inside forEach callback
Better solution 2: use filter then use sort
The best solution: use reduce and produce new array inside it(or two arrays like you do here and then concatenate them)